### ISSUE TITLE
Planet dock should be a sphere and not a hollow one.

### ISSUE THREAD

**Issue by royfalk** [vegastrike/Vega-Strike-Engine-Source#568]
When you come near a planet, docking only occurs at a specific, thin layer. 
If you pass that layer, you can't dock. 
This should not be the case. For playability, the dock should be spherical and extend from the center of the planet and due the edge of SPEC or near it. 
When we introduce cities, we can have their dock extend up in a box to this point.

**Comment by BenjamenMeyer** [comment]
yes. I'd also like to have the dock space farther out than it currently is. WC & Privateer also had it at 10,000 KM out versus our 1,000 KM.

**Comment by evertvorster** [comment]
I don't particularly mind the docks being so close to the planets, but it
does seem weird that the docks can be busy.
It seems that the docks being so close to the planet is a method to try and
slow down gameplay...

Kind regards,
Evert Vorster
Awesome Chapters Tours
http://www.awesomechapters.com
Tel: +264 (0) 811477690

**Comment by Loki1950** [comment]
Re: WC & Privateer they do not use manual docking they invoke a config file option called auto-docking.

**Comment by BenjamenMeyer** [comment]
> Re: WC & Privateer they do not use manual docking they invoke a config file option called auto-docking. 

They should allow it.

In the original games:
- WC did auto-dock only
- Privateer had auto-dock but also allowed manual dock, which was useful to escape enemies

**Comment by evertvorster** [comment]
Yeah, I did mention the busy docks in this bug report as well, but this bug report is about the shape of the dock.

**Comment by Loki1950** [comment]
That behaviour in Privateer is toggled on/off by the Auto Dock option in the settings app which sets it in the config file.

**Comment by royfalk** [comment]
Fixed by docking refactor. Closing.
